components/sandbox/toolchain-host-operator/base/proxy/proxyplugin.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: ProxyPlugin
+metadata:
+  name: tekton-results
+  namespace: toolchain-host-operator
+spec:
+  openShiftRouteTargetEndpoint:
+    # see https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/base/api-route.yaml
```

This URL doesn't resolve, did you mean this instead?
```suggestion
    # see https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/api-route.yaml
```